app/controllers/users_controller.rb

```diff
@@ -1,4 +1,26 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params) # Not the final implementation!
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
+                                 :password_confirmation)
```

Comment:
Please check your indentation settings in RubyMine.  We'd prefer this style:

```ruby
params.require(:user).permit(
  :name,
  :email,
  :password,
  :password_confirmation
)
```

Or
```ruby
params
  .require(:user)
  .permit(
    :name,
    :email,
    :password,
    :password_confirmation
  )
```